internal/duo/const.go

```diff
@@ -0,0 +1,16 @@
+package duo
+
+// Duo Methods.
+const (
+	// Push Method - The device is activated for Duo Push.
+	Push = "push"
+	// OTP Method - The device is capable of generating passcodes with the Duo Mobile app.
+	OTP = "mobile_otp"
+	// Phone Method - The device can receive phone calls.
+	Phone = "phone"
+	// SMS Method - The device can receive batches of SMS passcodes.
+	SMS = "sms"
+)
+
+// PossibleMethods is the set of all possible Duo 2FA methods.
+var PossibleMethods = []string{Push} // OTP, Phone, SMS
```

Comment:
Maybe this will be answered later in the PR, but why is this just `Push` instead of `Push, OTP, Phone, SMS`?

Reply:
Because for now we only want to allow pre-auth and enrolment, I think there's some other visual changes we will need to introduce into the frontend to accept other Duo capabilities.

Reply:
SGTM!